Project2015To2017/Analysis/Diagnostics/W021SystemNuGetPackagesDiagnostic.cs

```diff
@@ -143,7 +143,7 @@ public W021SystemNuGetPackagesDiagnostic() : base(21)
 			"System.IO.IsolatedStorage",
 		};
 
-		public StringComparison Comparison = ExtensionMethods.BestAvailableStringIgnoreCaseComparison;
+		public readonly StringComparison Comparison = Extensions.BestAvailableStringIgnoreCaseComparison;
```

Comment:
Can you make this a property?

Reply:
The whole point was to cache the result, I dunno why it is public though. Will make `private` in next PR, what do you think?

Reply:
It’s a field so I would never want that in the public API, if we can make it private that solves it.